spec/support/active_record.rb

```diff
@@ -0,0 +1,19 @@
+ActiveRecord::Base.establish_connection(
+  adapter: "sqlite3",
+  database: "#{ File.expand_path(File.join(File.dirname(__FILE__), '..')) }/db/test.sqlite3"
+)
+
+ActiveRecord::Base.connection.execute("DROP TABLE IF EXISTS 'posts'")
```

Comment:
No built in handling for actual fixtures? Or this just prevents the gem from loading un-necessary gems?